### PR TITLE
YOLOV2

### PR DESCRIPTION
**Pull request by mridul911**

### CODE REVIEW

**Comment by nirmal-suthar** [comment]
can you rearrange model in following files:
- data.py
- model.py
- main.py
- utils.py

Also, add a script for downloading and unpacking dataset (refer https://github.com/nirmal-suthar/dataloader/blob/master/celebA(multiLabel)/download.sh)

**Comment by mridul911** [comment]
rearranged the model

File: YOLOV2/README.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
+## Dataset
+```
+> VOC2012 dataset
+wget http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar
+```
+```
+> Unzippping Dataset
+tar -xvf  'VOCtrainval_11-May-2012.tar'
+```
```
Comment (nirmal-suthar) [inline YOLOV2/README.md:9]:
```suggestion
```bash
$ wget http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar
$ tar -xvf  'VOCtrainval_11-May-2012.tar'
```
```

File: YOLOV2/README.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+```
+> Also download weights 
+wget https://pjreddie.com/media/files/yolov2.weights
+```
```
Comment (nirmal-suthar) [inline YOLOV2/README.md:13]:
```suggestion
Also download weights
```bash 
$ wget https://pjreddie.com/media/files/yolov2.weights
```
```

File: YOLOV2/README.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
+## Dataset
```
Comment (nirmal-suthar) [inline YOLOV2/README.md:1]: Add titile

File: YOLOV2/README.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+```
+## Usage
+```bash
+$ python3 main.py --epochs 50 --batch_size 32
+```
+> **_NOTE:_** on Colab Notebook use following command:
+```python
+!git clone link-to-repo
+%run main.py --epochs 50 --batch_size 32
+```
```
Comment (nirmal-suthar) [inline YOLOV2/README.md:22]: add contributed by section

**Comment by nirmal-suthar** [comment]
- Now, you can remove ipynb book 
- also, rename folder to YOLOv2_TensorFlow
- make folder named assets and move all display files in that folder

**Pull request closed without merging**